modules/data-entry/src/main/frontend/src/questionnaireEditor/AnswerOptions.jsx

```diff
@@ -116,48 +119,39 @@ let AnswerOptions = (props) => {
         <Typography variant="subtitle2">Answer options:</Typography>
       </Grid>
       <Grid item xs={8}>
+      { deletedOptions.map((value, index) =>
+        <input type='hidden' name={`${value['@path']}@Delete`} value="0" key={value['@path']} />
+      )}
       { options.map((value, index) =>
-        <React.Fragment key={value['@path']}>
+        <React.Fragment key={value.value}>
           <input type='hidden' name={`${value['@path']}/jcr:primaryType`} value={'lfs:AnswerOption'} />
+          <input type='hidden' name={`${value['@path']}/label`} value={value.label} />
+          <input type='hidden' name={`${value['@path']}/value`} value={value.value} />
           <TextField
-            name={`${value['@path']}/value`}
-            defaultValue={value.value}
+            className={classes.answerOptionInput}
+            defaultValue={value.label? value.value + " = " + value.label : value.value}
+            onChange={(event) => { updateOption(index, event); }}
             multiline
             />
-          <IconButton onClick={() => { deleteOption(value) }}>
+          <IconButton onClick={() => { deleteOption(index); }} className={classes.answerOptionDeleteButton}>
             <CloseIcon/>
           </IconButton>
         </React.Fragment>
       )}
-      { newUUID.map((value, index) =>
-        <React.Fragment key={value}>
-          <input type='hidden' name={`${path}/${newValue[index]}/jcr:primaryType`} value={'lfs:AnswerOption'} />
-          <TextField
-            name={`${path}/${newValue[index]}/value`}
-            value={newValue[index]}
-            onChange={(event) => { updateInsertedOption(index, event); }}
-            multiline
-            />
-          <IconButton onClick={(event) => { deleteInsertedOption(index, event) }}>
-            <CloseIcon />
-          </IconButton>
-        </React.Fragment>
-      )}
-      { deletedOptions.map((value, index) =>
-        <input type='hidden' name={`${value['@path']}@Delete`} value="0" key={value['@path']} />
-      )}
       <TextField
+        fullWidth
         value={tempValue}
-        helperText='Press ENTER to add a new line'
-        onChange={(event) => { setTempValue(event.target.value); }}
-        onBlur={(event) => { handleInputOption(event); }}
+        error={isDuplicate}
+        helperText={isDuplicate ? 'duplicated value or label' : 'value OR value=label (e.g. F=Female)'}
```

Comment:
We should keep the instructions `Press ENTER to add a new line`. Something like this might work:
```suggestion
        helperText={isDuplicate ? 'duplicated value or label' : [<div>value OR value=label (e.g. F=Female)</div>, <div>Press ENTER to add a new line</div>]}
```